auth/net_tools.go

```diff
@@ -72,7 +79,7 @@ func auditUserLogAction(central *ImqsCentral, req *httpRequest, userId authaus.U
 		if user, err := central.Central.GetUserFromUserId(authaus.UserId(actorUserId)); err == nil {
```

Comment:
If I look at this line, it kinda feels like a duplication of line 52. But it is probably here to catch a loose end if the `req.token.UserId` is populated.